rollup/internal/controller/watcher/batch_proposer_test.go

```diff
@@ -227,15 +216,6 @@ func testBatchProposerCodecv1Limits(t *testing.T) {
 			expectedBatchesLen:         1,
 			expectedChunksInFirstBatch: 1,
 		},
-		{
```

Comment:
why are we removing these tests? We should still have support for height-based forks, no?

Reply:
added in [887d2b1](https://github.com/scroll-tech/scroll/pull/1397/commits/887d2b1f851049a91b652206f648ca5b7302313c).